### DISCUSSION TITLE
Windows installer issues

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [tauri-apps/tauri#4989] [General]
The bundled app has some issues for Windows. When installing the app it creates a desktop shortcut without even asking. (I guess this could be configured - Are there any examples on how to do this?)

Also, there is a problem, the resolution for the icon used by default isn't the best. Which icon is used for this?
At the same time, and as you can see on the image, it creates a desktop.ini file during the installation.

![1](https://user-images.githubusercontent.com/8037559/185802271-972ba654-7e51-4e8d-a099-faefdf248d16.JPG)

**Comment by FabianLars** [comment]
> When installing the app it creates a desktop shortcut without even asking. 

The upcoming NSIS installer will ask before doing this. For the current one you can change the wix file Tauri uses and disable that in there, or, if possible add a new checkbox to let the user decide (should be possible but idk how), see https://tauri.app/v1/guides/building/windows#replacing-the-installer-code-with-a-custom-wix-file

> Also, there is a problem, the resolution for the icon used by default isn't the best. Which icon is used for this?

The icon.ico file. `tauricon` current ico output is not the best, but the default Tauri icons were manually generated if i remember correctly and should contain the correct sizes (an ico size contains multiple layers and Windows decides which one to use depending on the context).

> At the same time, and as you can see on the image, it creates a desktop.ini file during the installation.

No idea where it comes from but i can't remember any complains about that 🤔 
Buuut this is only an issue if you changed the system settings to also show system protected files, soooo in my opinion not a huge problem (i don't think we could even do anything about this)

**Comment by amrbashir** [comment]
`desktop.ini` is autogenerated by Windows, read more about it here https://www.thewindowsclub.com/desktop-ini-file-windows

**Reply by MrAmericanMike** [reply]
> `desktop.ini` is autogenerated by Windows, read more about it here https://www.thewindowsclub.com/desktop-ini-file-windows

Well I mentioned because I never saw it happening with an Electron app. But I guess each installer handles things in a different way.

Thanks, will take a look at that link.

**Comment by apalacean-panaya** [comment]
@MrAmericanMike did you succeed in not creating a desktop shortcut? If yes, how?

**Reply by MrAmericanMike** [reply]
I haven't played with/tested Tauri much more, so I didn't have the chance to test this again.
